lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -207,6 +206,24 @@ def initialize
       @log = $log # rubocop:disable Style/GlobalVars
     end
 
+    # Set up regexp patterns to parse tags and logs.
+    def setup_regexp_patterns
```

Comment:
Oh, I forgot to move this to where it should belong. Just moved. Also rename it to be `set_regexp_patterns` instead of `setup_regexp_patterns`.